pkg/pillar/cmd/baseosmgr/baseosmgr.go

```diff
@@ -502,104 +502,111 @@ func initializeGlobalConfigHandles(ctx *baseOsMgrContext) {
 
 	// Look for global config such as log levels
 	subGlobalConfig, err := pubsub.Subscribe("", types.GlobalConfig{},
-		false, ctx)
+		false, ctx, &pubsub.SubscriptionOptions{
```

Comment:
Do we need to pass Ptr for Subscription options? If passign values for Read purposes, we are just passign it by value ( Not Ptr ) even though it is a struct.